q2_feature_classifier/classifier.py

```diff
@@ -170,8 +170,7 @@ def classify_sklearn(reads: DNAIterator, classifier: Pipeline,
     name='Pre-fitted sklearn-based taxonomy classifier',
     description='Classify reads by taxon using a fitted classifier.',
     parameter_descriptions={'confidence': 'Confidence threshold for limiting '
-                            'taxonomic depth. Negative value disables '
-                            'Currently experimental. USE WITH CAUTION',
+                            'taxonomic depth. Negative value disables.',
```

Comment:
Can you can this to: 

```
'taxonomic depth. Provide -1 to disable '
'confidence calculation, or 0 to calculate '
'confidence but not apply it to limit the '
'taxonomic depth of the assignments.'
```

I know that you can provide other negative values, but I think we'll get fewer support questions about this if we are just very specific about what the value should be.